source/features/bugs-tab.tsx

```diff
@@ -39,7 +39,7 @@ async function init(): Promise<void | false> {
 		return false;
 	}
 
-	const issuesTab = (await elementReady('.pagehead [data-hotkey="g i"]'))?.parentElement;
+	const issuesTab = (await elementReady('[data-hotkey="g i"]'))?.parentElement;
```

Comment:
Unfortunately, they are not unique:

![Screenshot_2020-06-10_13-43-27](https://user-images.githubusercontent.com/202916/84263770-5f09c380-ab20-11ea-8489-f38d4a346a23.png)